Gordon360/Controllers/NewsController.cs

```diff
@@ -198,5 +199,27 @@ public ActionResult<StudentNewsViewModel> EditPosting(int newsID, [FromBody] Stu
             var result = _newsService.EditPosting(newsID, studentNewsEdit);
             return Ok(result);
         }
+
+        /// <summary>
+        ///  Approve a news posting in the database
+        /// </summary>
+        /// <param name="newsID">The id of the news item to approve</param>
+        /// <param name="newsStatusAccepted">The accept status that will apply to the news item</param>
+        /// <returns>The approved news item</returns>
+        /// <remarks>The news item must not be expired and must be unapproved</remarks>
+        [HttpPut]
+        [Route("{newsID}/accepted")]
+        // only SNAdmin is authorized to use this route
+        // Can't use SYB UPDATE operation, because the author is not authorized to approve the post
+        public ActionResult<StudentNewsViewModel> ApprovePosting(int newsID, [FromBody] bool newsStatusAccepted)
+        {
+            if (user_groups.Contains(AuthGroup.SiteAdmin) || user_groups.Contains(AuthGroup.NewsAdmin))
```

Comment:
Let's not allow site admins to approve news admin. We've been trying to get away from giving site admins global permissions.
```suggestion
            if (user_groups.Contains(AuthGroup.NewsAdmin))
```